File: testprojects/c#/benchmark/domain/locationbased/foursquare/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.socialmedia.locationbased.foursquare;

namespace CSharpBenchmark.domain.locationbased.foursquare
{
    //Functional requirement 3.2
    //Test case 124: Class domain.locationbased.foursquare.Event is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.EventDAO 
    //Result: FALSE
    public class Event
    {
        //FR5.5	
        private EventDAO dao;

        public Event()
        {
            //FR5.2
            Console.WriteLine(dao.name);
        }
    }
}

File: testresources/java/benchmark_2012/domain/locationbased/latitude/Event.java
package domain.locationbased.latitude;

import infrastructure.socialmedia.locationbased.latitude.EventDAO;

//Functional requirement 3.2
//Test case 123: Class domain.locationbased.latitude.Event is not allowed to use class infrastructure.socialmedia.locationbased.foursquare.EventDAO  
//Result: TRUE
public class Event {
	//FR5.5	
	private EventDAO dao;

	public Event(){
		//FR5.2
		System.out.println(dao.name);
	}
}